components/case/pilotStepHeader/index.tsx

```
@@ -21,16 +26,37 @@ function PurePilotStepHeader(props: PilotStepHeaderProps) {
   };
 
-  const handleBtnGotoOfficialClick = () => {
-    console.log('handleBtnGotoOfficialClick');
-  };
+  // const handleBtnGotoOfficialClick = () => {
+  //   console.log('handleBtnGotoOfficialClick');
+  // };
 
   const handleBtnViewClick = () => {
     console.log('handleBtnViewClick');
+    if (!!pilotInfo?.tabInfo?.url) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)